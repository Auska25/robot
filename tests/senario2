MOVE
LEFT
LEFT
MOVE
REPORT
MOVE
RIGHT
REPORT
MOVE
PLACE 0,3,WEST
MOVE
LEFT
REPORT
LEFT
MOVE
MOVE
MOVE
REPORT
RIGHT
MOVE
REPORT
QUIT
