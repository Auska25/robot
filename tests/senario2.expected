SYSTEM START
ROBOT HASN'T BE PLACED SUCCESSFULLY
ROBOT HASN'T BE PLACED SUCCESSFULLY
0,3,SOUTH
3,3,EAST
3,2,SOUTH
SYSTEM EXIT
